src/Core/maps/src/Core/IMapPin.cs

```diff
@@ -23,6 +23,11 @@ public interface IMapPin : IElement
 		/// </summary>
 		Location Location { get; }
 
+		/// <summary>
+		/// Tag of this pin
+		/// </summary>
+		object? Tag { get; }
```

Comment:
I am wondering if this needs to go all the way to core? This is very much a XAML type of feature... maybe?

Reply:
I mean that this Tag property is maybe not meant to go all the way to Core as it does not do anything. It can just live in Controls and not on the IMapPin interface. But again, this was more a discussion point than an action item.

If you have feelings either way, then let us know down in the comments below because I have subscribed and hit the bell! :)